src/java.base/share/classes/com/sun/crypto/provider/DHPrivateKey.java

```diff
@@ -273,10 +275,10 @@ public DHParameterSpec getParams() {
         }
     }
 
-    private void parseKeyBits() throws InvalidKeyException {
```

Comment:
This method is only called once inside the constructor, considering that this method is kind of trivial, maybe you can just move the impl to the constructor and remove this method.